### PR TITLE
Verbose mode

### PR DESCRIPTION
**Pull request by tapajos**
A good unix command include a verbose mode.

### CODE REVIEW

File: lib/kumade/runner.rb [hunk 1]
```diff
@@ -38,6 +38,10 @@ module Kumade
           options[:cedar] = cedar
         end
 
+        opts.on_tail("-vv", "--verbose", "Print what kumade is doing") do
```
Comment (gabebw) [inline lib/kumade/runner.rb]: Hm, could you change this to "-v" and change `--version` to only accept the long option (not `-v`)? I think people will use verbose mode a lot more than they check the version, and `-vv` seems wrong to me.

Comment (gabebw) [inline lib/kumade/runner.rb]: It's not your fault, you just pointed out a command-line design mistake on my part :)

**Comment by tapajos** [comment]
Ready to merge.

File: spec/kumade/runner_spec.rb [hunk 2]
```diff
@@ -73,4 +73,15 @@ describe Kumade::Runner do
       $stdout.puts "Hello, you can see me!"
     end
   end
+  it 'prints everything when verbose is true' do
```
Comment (gabebw) [inline spec/kumade/runner_spec.rb]: newline before this line, please :D

File: lib/kumade/runner.rb [hunk 3]
```diff
@@ -70,5 +74,13 @@ module Kumade
     def self.pretending?
       @options[:pretend]
     end
+    
+    def self.verbose?
```
Comment (gabebw) [inline lib/kumade/runner.rb]: Can you please add a little test for verbose? and print_output?

**Comment by gabebw** [comment]
Mostly looks good, I just have a couple of comments before I can merge it.

**Comment by tapajos** [comment]
Ready?

**Comment by tapajos** [comment]
Ready again. Development is happening really fast

**Comment by tapajos** [comment]
Ready again

**Pull request merged**